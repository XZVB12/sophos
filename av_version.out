SAVScan virus detection utility
Copyright (c) 1989-2017 Sophos Limited. All rights reserved.

System time 18:00:57, System date 16 June 2017

Product version           : 5.34.0
Engine version            : 3.68.0
Virus data version        : 5.40
User interface version    : 2.03.068
Platform                  : Linux/AMD64
Released                  : 30 May 2017
Total viruses (with IDEs) : 13427613
