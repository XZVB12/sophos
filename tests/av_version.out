SAVScan virus detection utility
Copyright (c) 1989-2017 Sophos Limited. All rights reserved.

System time 19:51:11, System date 10 September 2017

Product version           : 5.38.0
Engine version            : 3.69.2
Virus data version        : 5.43
User interface version    : 2.03.069
Platform                  : Linux/AMD64
Released                  : 22 August 2017
Total viruses (with IDEs) : 13985024
